### DISCUSSION TITLE
A UI to show the combined resources for the party and Frosthaven

### DISCUSSION THREAD

**Discussion by tomas-pals** [Lurkars/gloomhavensecretariat#403] [Ideas]
I find when playing Frosthaven that we are often wondering what building(s) we can build as well as which herbs we have combined. Clicking in on each character sheet to see what everybody has is not optimal as you forget what each one had easily.

My idea was to have a dialog similar to the one when you build a building except it would show the inventory of all resources (both material and herb)
![CleanShot 2023-09-25 at 21 43 13@2x](https://github.com/Lurkars/gloomhavensecretariat/assets/2796248/3c27e157-e3f3-4c96-a4a0-a7392bcf2d45)

I would then place a button in the _Frosthaven supply_ section of the Party sheet to open this UI, perhaps next to _Move to Frosthaven supply_
<img width="453" alt="CleanShot 2023-09-25 at 21 43 40@2x" src="https://github.com/Lurkars/gloomhavensecretariat/assets/2796248/2b88b8f6-c843-42ec-8524-605a9ed6bfba">

**Comment by Lurkars** [comment]
I will think about it. Until now, buildings to craft should be available when you use the tracking for building, unlock the corresponding buildings and you should see which are buildable/upgradeable!

**Reply by tomas-pals** [reply]
Yes that's correct and it works as intended. We've just had a lot of times where players are wondering what to craft or whether we should build a building with the available resources and it's nice to just be able to open a dialog showing all of the info so anyone can see it without having to ask the one responsible for the computer. Look forward to hearing your thoughts :)

**Comment by Lurkars** [comment]
First draft now in [v0.77.13](https://github.com/Lurkars/gloomhavensecretariat/releases/tag/v0.77.13). By now, it's a bit hidden, just click on the "Frosthaven Supply" Title on the campaign sheet to see an overview. If this is what you have in mind, we can discuss some improvements and better UX.
